### PR TITLE
Switch to use ft-poller

### PR DESCRIPTION
**Pull request by i-like-robots**
Merge after #56 =]

### CODE REVIEW

**Comment by andygout** [comment]
Can you remind me of the benefit of polling from what is quite a small amount of data?

File: server/app.js [hunk 1]
```diff
@@ -31,6 +31,4 @@ specialPagesPoller.init();
 // Start the app
 const port = process.env.PORT || 3001;
 
-module.exports.listen = app.listen(port, function () {
-	console.log('Listening on ' + port);
```
Comment (andygout) [inline server/app.js]: Not useful?

**Comment by i-like-robots** [comment]
the data doesn't change too often so adding the overhead of requesting it each time would be both wasteful and slow. It may also help in the event of a Bertha outage (Which is bronze) because we can keep using the stale data.

**Comment by andygout** [comment]
👍 

Ah, thanks - have just read over the `ft-poller` documentation as well.

File: server/app.js (hunk 1 above)
Comment (i-like-robots) [inline server/app.js]: The port is output by n-express already =]

**Pull request merged**